api/.env

```diff
@@ -11,7 +11,7 @@ REDIS_HOST=redis1
 UAZ_URL=http://linking.cs.arizona.edu/v1
 UAZ_THRESHOLD=0.6
 UAZ_HITS=10
-CAUSEMOS_DEBUG=true
+CAUSEMOS_DEBUG=false
 CAUSEMOS_USER=worldmodelers
 CAUSEMOS_PWD=world!
 CAUSEMOS_IND_URL=https://causemos.uncharted.software/api/maas
```

Comment:
Here CAUSEMOS_IND_URL is not testmos- does that matter? (Will exec into my running api container and check now).

Reply:
Verified that on my system this created a docker-compose.yaml file with the prod causemos value (not testmos).